src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -3,6 +3,7 @@ import * as React from 'react';
 import {useRecoilState, useRecoilValue} from 'recoil';
 import {useRoute} from '@react-navigation/native';
 
+import {Platform} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Platform` from 'react-native' is a new addition to handle platform-specific logic. This change seems appropriate given the context.
```diff
+ import {Platform} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->